README.md

```diff
@@ -155,21 +155,30 @@ and in your `configuration.nix`
 {inputs, pkgs, ... }:
 let
   inherit (pkgs.stdenv.hostPlatform) system;
-  umu = inputs.umu.packages.${system}.umu.override {
-    version = inputs.umu.shortRev;
-    truststore = true;
-    cbor2 = true;
-  };
+  umu-launcher = inputs.umu.packages.${system}.default;
 in
 {
-  environment.systemPackages = [ umu ];
+  environment.systemPackages = [ umu-launcher ];
 }
 ```
-> [!NOTE]
-> truststore and cbor2 (for delta updates) are optional dependency which are enabled by default if you want to keep it that way you can remove the `truststore = true; cbor2 = true;` part
+
+> [!TIP]
+> You can override `withTruststore` and/or `withDeltaUpdates` to disable optional dependencies.
```

Comment:
@LovingMelody should we have an `extraPkgs`/`extraLibraries` example in the README too?

Maybe it's a bit out of scope for this PR, but if you have an example and/or some wording you'd suggest I can add something in an additional commit.